2021-07-01 11:27:13,713 INFO    MainThread:15296 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 11:27:13,713 INFO    MainThread:15296 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 11:27:13,713 INFO    MainThread:15296 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 11:27:13,713 INFO    MainThread:15296 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_112713-1uw6d2ob\logs\debug.log
2021-07-01 11:27:13,713 INFO    MainThread:15296 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_112713-1uw6d2ob\logs\debug-internal.log
2021-07-01 11:27:13,760 INFO    MainThread:15296 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F6BB83CAC8>
2021-07-01 11:27:13,760 INFO    MainThread:15296 [wandb_init.py:init():370] calling init triggers
2021-07-01 11:27:13,760 INFO    MainThread:15296 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 11:27:13,760 INFO    MainThread:15296 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 1zu8k6n2
2021-07-01 11:27:13,760 INFO    MainThread:15296 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Captstone-Project/1zu8k6n2
2021-07-01 11:27:13,760 INFO    MainThread:15296 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-07-01 11:27:13,775 INFO    MainThread:15296 [wandb_run.py:_restore():1566] restore
2021-07-01 11:27:14,755 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 102982
  total_bytes: 102982
}

2021-07-01 11:27:14,878 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 102982
  total_bytes: 102982
}

2021-07-01 11:27:15,160 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 102982
  total_bytes: 102982
}

2021-07-01 11:27:15,456 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 102982
  total_bytes: 107052
}

2021-07-01 11:27:15,801 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 102982
  total_bytes: 107052
}

2021-07-01 11:27:15,910 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:16,019 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:16,129 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:16,238 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:16,348 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:16,457 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:16,566 INFO    MainThread:15296 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107052
  total_bytes: 107052
}

2021-07-01 11:27:17,675 INFO    MainThread:15296 [wandb_run.py:_show_summary():1871] rendering summary
2021-07-01 11:27:17,691 INFO    MainThread:15296 [wandb_run.py:_show_history():1909] rendering history
2021-07-01 11:27:17,691 INFO    MainThread:15296 [wandb_run.py:_show_files():1938] logging synced files
2021-07-01 11:27:17,691 INFO    MainThread:15296 [wandb_init.py:init():419] starting backend
2021-07-01 11:27:17,691 INFO    MainThread:15296 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-07-01 11:27:17,706 INFO    MainThread:15296 [backend.py:ensure_launched():135] starting backend process...
2021-07-01 11:27:17,863 INFO    MainThread:15296 [backend.py:ensure_launched():140] started backend process with pid: 18912
2021-07-01 11:27:17,863 INFO    MainThread:15296 [wandb_init.py:init():424] backend started and connected
2021-07-01 11:27:17,878 INFO    MainThread:15296 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-07-01 11:27:17,878 INFO    MainThread:15296 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-01 11:27:17,878 INFO    MainThread:15296 [wandb_init.py:init():472] updated telemetry
2021-07-01 11:27:17,894 INFO    MainThread:15296 [wandb_init.py:init():491] communicating current version
2021-07-01 11:27:20,870 INFO    MainThread:15296 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.10.33 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-01 11:27:20,870 INFO    MainThread:15296 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-01 11:27:21,323 INFO    MainThread:15296 [wandb_init.py:init():529] starting run threads in backend
2021-07-01 11:27:21,808 INFO    MainThread:15296 [wandb_run.py:_console_start():1624] atexit reg
2021-07-01 11:27:21,808 INFO    MainThread:15296 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-07-01 11:27:21,808 INFO    MainThread:15296 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-07-01 11:27:21,808 INFO    MainThread:15296 [wandb_run.py:_redirect():1559] Redirects installed.
2021-07-01 11:27:21,808 INFO    MainThread:15296 [wandb_init.py:init():554] run started, returning control to user process
2021-07-01 11:27:23,183 INFO    MainThread:15296 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'outlier_candidates') {'id': 'wandb/outliers/v1', 'historyFieldSettings': {'key': 'outlier_candidates', 'x-axis': '_step'}} None
2021-07-01 11:27:50,170 INFO    MainThread:15296 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'learning_curve') {'id': 'wandb/learning_curve/v1', 'historyFieldSettings': {'key': 'learning_curve', 'x-axis': '_step'}} None
2021-07-01 11:27:54,848 INFO    MainThread:15296 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'residuals') {'id': 'wandb/residuals_plot/v1', 'historyFieldSettings': {'key': 'residuals', 'x-axis': '_step'}} None
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_112806-3qbuaf9k\logs\debug.log
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_112806-3qbuaf9k\logs\debug-internal.log
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_init.py:init():370] calling init triggers
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 1uw6d2ob
2021-07-01 11:28:06,921 INFO    MainThread:15296 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Captstone-Project/1uw6d2ob
